.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,21 @@ Addresses issue #
 - [ ] My code has proper inline documentation.
 - [ ] I have added a QA Brief on the issue linked above.
 - [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).
+
+---------------
```

Comment:
Let's also add something like this here, similar to how we have in issues:
```suggestion
---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._
```

Maybe "code reviewers" would be more precise here than saying "moderators", but I don't have a strong opinion - I think the term "moderators" has the benefit that it clarifies that a 3P contributor shouldn't touch this section. Either works for me.

Reply:
Thanks. Either works for me too. Accepted your suggestion.